metricbeat/module/windows/perfmon/pdh_query_windows_test.go

```diff
@@ -87,3 +88,32 @@ func TestSuccessfulQuery(t *testing.T) {
 	assert.Nil(t, err)
 	assert.NotNil(t, list)
 }
+
+// TestInstanceNameRegexp tests regular expression for instance.
+func TestInstanceNameRegexp(t *testing.T) {
+	queryPaths := []string{`\SQLServer:Databases(*)\Log File(s) Used Size (KB)`, `\Search Indexer(*)\L0 Indexes (Wordlists)`,
+		`\Search Indexer(*)\L0 Merges (flushes) Now.`, `\NUMA Node Memory(*)\Free & Zero Page List MBytes`}
+	for _, path := range queryPaths {
+		matches := instanceNameRegexp.FindStringSubmatch(path)
+		if len(matches) == 2 {
```

Comment:
Maybe we can use assert.equal(t, 2, len(matches)) here instead of the if else statement? Same in TestObjectNameRegexp function.

Reply:
Actually assertions can be used as conditions too, so this if could be rewritten as something like this:
```
if assert.Len(t, matches, 2, "regular expression should match") {
  assert.Equal(t, matches[1], "*")
}
```